setup.py

```diff
@@ -28,53 +28,52 @@
         ]
     },
     install_requires=[
-        "googleapis-common-protos>=1.57",
-        "flyteidl>=1.10.0",
-        "wheel>=0.30.0,<1.0.0",
-        "pandas>=1.0.0,<2.0.0",
-        "pyarrow>=4.0.0,<11.0.0",
+        "adlfs",
```

Comment:
nit: Could we add a comment here? like
```suggestion
        # Please maintain an alphabetical order in the following list
        "adlfs",
```

Reply:
btw, fixed the unit tests here. https://github.com/flyteorg/flytekit/pull/1934

Reply:
Thanks for fixing the unit tests 👍 

I've added the suggested line in https://github.com/flyteorg/flytekit/pull/1931/commits/c7cbfff418530a2d7fc6f21d2bc47facdf9c512a

Reply:
@bstadlbauer thank you. one nit: there is a lint error, could you run `make fmt`, and push it again.